src/services/scenario/ScenarioService.js

```diff
@@ -37,9 +39,40 @@ function createScenario (organizationId, workspaceId, scenario) {
   });
 }
 
-function updateAndLaunchScenario (organizationId, workspaceId, scenariodId, scenarioParameters) {
+function formatScenarioParametersForApi (scenarioParameters) {
```

Comment:
Should be placed in utility file.
We will certainly reuse the logic here somewhere else :thinking: